src/lib/viewers/BaseViewer.js

```diff
@@ -989,6 +989,7 @@ class BaseViewer extends EventEmitter {
 
         if (this.areNewAnnotationsEnabled() && this.annotationControls) {
             this.annotator.addListener('annotations_create', this.handleAnnotationCreateEvent);
+            this.handleAnnotationsOnLoad();
```

Comment:
This seems likely to lead to a race condition, since we are assuming the fetch call from box-annotations has completed by this point. Should we emit a `annotations_fetch_success` event from box-annotations with a payload of the fetched annotations? Then we can pull the annotation we want out of that payload and scroll to it?

Reply:
Conrad eluded to this as well, I think that is a good approach as well.